airgun/entities/all_hosts.py

```diff
@@ -547,6 +548,72 @@ def disassociate_hosts(self, host_names, select_all_hosts=False):
         view = DisassociateHostsModal(self.browser)
         view.confirm_btn.click()
 
+    def select_any_number_of_hosts(
```

Comment:
Good idea with this helper 👍🏻 , I have the similar approach implemented right now locally, it would be also nice to use it everywhere in the file where it is possible to use it, but you don't have to do that in this PR as I have this change prepared locally as all_hosts.py files refactor I am planning to push in near future.